src/pilot/templates/static-ip-environment.yaml

```diff
@@ -84,6 +91,26 @@ parameter_defaults:
     - 192.168.170.31
     - 192.168.170.32
     - 192.168.170.33
+    
```

Comment:
I think it will be confusing to leave the NovaComputeIPs: entries in this file.  Can you remove them?  I think it should be safe to do.